code/modules/projectiles/guns/revolvers.dm

```diff
@@ -667,6 +668,18 @@
 	current_mag = /obj/item/ammo_magazine/internal/revolver/mateba/impact
 	map_specific_decoration = TRUE
 
+/obj/item/weapon/gun/revolver/mateba/special
+	name = "\improper Mateba autorevolver special"
+	desc = "An old, heavily modified version of the Mateba Autorevolver. It sports a smooth wooden grip, and a much larger barrel to it's unmodified counterpart. It's clear that this weapon has been cared for over a long period of time."
+	icon_state = "cmateba_special"
+	item_state = "cmateba_special"
+	current_mag = /obj/item/ammo_magazine/internal/revolver/mateba/impact
+	can_change_barrel = FALSE
+
+/obj/item/weapon/gun/revolver/mateba/special/set_gun_config_values()
+	..()
+	accuracy_mult = BASE_ACCURACY_MULT + HIT_ACCURACY_MULT_TIER_4
```

Comment:
Why does this variant of it have a higher accuracy mult than the standard one?

Reply:
It doesn't use the barrel extension